dm3-names/contracts/l2/Dm3NameRegistrar.sol

```diff
@@ -0,0 +1,192 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
```

Comment:
Is there a reason for using solidity 0.8.17 instead of a more current version? We are at 0.8.25 right now.

Reply:
All the other projects in the monorepo are using  0.8.17 and I don't want to use multiple different solidity version

Reply:
I set the version to 0.8.25 for the entire repo, since OZ v5 requires at least  0.8.20